packages/amplify-category-function/src/provider-utils/awscloudformation/utils/layerParams.ts

```diff
@@ -0,0 +1,17 @@
+import { FunctionRuntime, ProviderContext } from 'amplify-function-plugin-interface';
+
+export type LayerParameters = {
+  layerName: string;
+  layerPermissions: Permissions;
+  runtimes: FunctionRuntime[];
+  providerContext: ProviderContext;
+  authorizedAccountIds?: string;
+  authorizedOrgId?: string;
+};
+
+export enum Permissions {
+  private = 'private',
+  public = 'public',
+  awsAccs = 'aws-accs',
```

Comment:
nit make the string and the enum the same (unless these are actual CFN values in which case name the enum CFNPermissions)

Reply:
fixed